message_app/__init__.py

```diff
@@ -1,73 +1,140 @@
+import sys
+import os
+sys.path.append(os.getcwd())
```

Comment:
If this line is to help run the test, we should have it in the test repo, somewhere in the test file?